### PR TITLE
Update README.MD

### PR DESCRIPTION
**Pull request by ran88dom99**
api database

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -26,6 +26,7 @@ It's always been sad to see a service shutdown, or hardware stop renew, here is
 - [Flowing Data](http://flowingdata.com/category/self-surveillance/) - Self surveillance and quantification news.
 - [Biohack Stack](https://biohackstack.com/) - Biohacker community with "stacks" of famous scientists.
 - [Wearables Database](http://vandrico.com/wearables/) - Awesome list of wearables available per body part.
+- [Database of Health Research related APIs](http://www.health2api.com/?category_name=api&category__and%5B%5D=7854&category__and%5B%5D=0&category__and%5B%5D=0&posts_per_page=50&submit=filter) - More for researchers.
```
Comment (agucova) [inline README.MD:29]: Maybe it would be better to strip the query from the URL.

Comment (ran88dom99) [reply README.MD:29]: That leads to a list of reaserch stufff not QS.

Comment (agucova) [reply README.MD:29]: Your query seems to result in 5 entries one of which is "Restaurant health inspection scores" and another "Supermarket API Is The First Robust Open Source API For The Grocery Industry."

I'm not sure if the list is really that relevant for QS.

**Comment by agucova** [comment]
Maybe we could take the particularly relevant entries and add them instead?

File: README.MD (hunk 1 above)
Comment (ran88dom99) [reply README.MD:29]: Its great for researchers and developers. All the food apps could really benefit from informing their users of health inspection scores. Same with the supermarket API.

Comment (ran88dom99) [reply README.MD:29]: A "For developers" sections could be a thing.

Comment (agucova) [reply README.MD:29]: I agree that it's great for researchers and developers, but I fail to see how it's relevant to the topic at hand.

Health inspection scores are important, but bare not relation at all with the Quantified Self movement. In the same way, Supermarket API may be great for the supermarket industry, but I can't say the same for QS.

Comment (ran88dom99) [reply README.MD:29]: Ok you are right. Or at least its very outdated.

**Comment by ran88dom99** [comment]
> 
> 
> Maybe we could take the particularly relevant entries and add them instead?

Yes.

**Comment by woop** [comment]
Thanks for the discussion, closing for now. You will submit new PRs @ran88dom99?

**Pull request closed without merging**